specification/cost-management/resource-manager/Microsoft.CostManagement/stable/2020-06-01/costmanagement.json

```diff
@@ -0,0 +1,3287 @@
+        }
+      },
+      "required": [
+        "from",
+        "to"
+      ]
+    },
+    "ExportExecutionListResult": {
+      "description": "Result of listing the execution history of an export.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "A list of export executions.",
+          "type": "array",
+          "readOnly": true,
+          "items": {
+            "$ref": "#/definitions/ExportExecution"
+          }
+        }
+      }
+    },
+    "ExportExecution": {
+      "description": "An export execution.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/Resource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/ExportExecutionProperties",
+          "title": "Export execution properties"
+        }
+      }
+    },
+    "ExportExecutionProperties": {
+      "description": "The properties of the export execution.",
+      "type": "object",
+      "properties": {
+        "executionType": {
+          "description": "The type of the export execution.",
+          "type": "string",
+          "enum": [
+            "OnDemand",
+            "Scheduled"
+          ],
+          "x-ms-enum": {
+            "name": "ExecutionType",
+            "modelAsString": true
+          }
+        },
+        "status": {
+          "description": "The last known status of the export execution.",
+          "type": "string",
+          "enum": [
+            "Queued",
+            "InProgress",
+            "Completed",
+            "Failed",
+            "Timeout",
+            "NewDataNotAvailable",
+            "DataNotAvailable"
+          ],
+          "x-ms-enum": {
+            "name": "ExecutionStatus",
+            "modelAsString": true
+          }
+        },
+        "submittedBy": {
+          "description": "The identifier for the entity that executed the export. For OnDemand executions it is the user email. For scheduled executions it is 'System'.",
+          "type": "string"
+        },
+        "submittedTime": {
+          "description": "The time when export was queued to be executed.",
+          "type": "string",
+          "format": "date-time"
+        },
+        "processingStartTime": {
+          "description": "The time when export was picked up to be executed.",
+          "type": "string",
+          "format": "date-time"
+        },
+        "processingEndTime": {
+          "description": "The time when the export execution finished.",
+          "type": "string",
+          "format": "date-time"
+        },
+        "fileName": {
+          "description": "The name of the exported file.",
+          "type": "string"
+        },
+        "runSettings": {
+          "description": "The export settings that were in effect for this execution.",
+          "$ref": "#/definitions/CommonExportProperties"
+        },
+        "errorCode": {
```

Comment:
please place "code" and "message" inside an "error" object so it conforms to the standard error structure

Reply:
Done